.travis.yml

```diff
@@ -20,6 +20,9 @@ before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.28.1
 
 script:
+  # Babel doesn't build correctly with npm v2 (which comes bundled with Node v4)
+  # so we manually upgrade to a known working npm version.
+  - npm install --global npm@4
```

Comment:
It's because of the **giant** directory structures it produces. We do warn about something similar on the Babel site:

> Note: Running a Babel 6.x project using npm 2.x can cause performance problems because of the way npm 2.x installs dependencies. This problem can be eliminated by either switching to npm 3.x or running npm 2.x with the dedupe flag.

dedupe might work, but it was easiest to just upgrade npm. npm 5 is buggy so I stuck with 4.